src/viewer/scene/model/vbo/instancing/triangles/VBOInstancingTrianglesLayer.js

```diff
@@ -378,7 +378,7 @@ export class VBOInstancingTrianglesLayer {
             && !!textureSet
             && !!textureSet.colorTexture;
 
-        this._state.geometry = null;
+        // this._state.geometry = null;
```

Comment:
This removal seems potentially very impactful if there are places that depend on the property test, or by not freeing memory that would otherwise get released.
What would be a reason for the geometry to be kept as the `_state` property, even though the layer was being finalized?

Reply:
Agreed - better put this line back @ijlal99

Reply:
I have moved the indices and vertices to 'this._state' and put this line back again.